From 2e3be6781d18274fd5c4d656c258dc141a7fe26a Mon Sep 17 00:00:00 2001
From: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
Date: Thu, 25 May 2023 10:46:48 +0600
Subject: [PATCH 70/85] arm64: configs: vc4: enable rswitch support

- enable Marvel 1G phy drivers
- enable Marvel 2.5G phy driver as module
- enable rswitch2 driver as module

Signed-off-by: Nikita Yushchenko <nikita.yoush@cogentembedded.com>
---
 arch/arm64/configs/renesas-vc4_defconfig | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/arch/arm64/configs/renesas-vc4_defconfig b/arch/arm64/configs/renesas-vc4_defconfig
index ed722df6996b..de1513aa5d08 100644
--- a/arch/arm64/configs/renesas-vc4_defconfig
+++ b/arch/arm64/configs/renesas-vc4_defconfig
@@ -289,6 +289,9 @@ CONFIG_LAN865X=m
 # CONFIG_NET_VENDOR_RDC is not set
 # CONFIG_NET_VENDOR_REALTEK is not set
 CONFIG_RAVB=y
+CONFIG_RENESAS_RSWITCH2=m
+CONFIG_RENESAS_RSWITCH2_PLATF=y
+CONFIG_RENESAS_RSWITCH2_GWCA=y
 # CONFIG_NET_VENDOR_ROCKER is not set
 # CONFIG_NET_VENDOR_SAMSUNG is not set
 # CONFIG_NET_VENDOR_SEEQ is not set
@@ -305,6 +308,8 @@ CONFIG_RAVB=y
 # CONFIG_NET_VENDOR_VIA is not set
 # CONFIG_NET_VENDOR_WIZNET is not set
 # CONFIG_NET_VENDOR_XILINX is not set
+CONFIG_MARVELL_PHY=y
+CONFIG_MARVELL_88Q3344_ES_PHY=m
 # CONFIG_WLAN is not set
 CONFIG_NET_FAILOVER=y
 CONFIG_INPUT_FF_MEMLESS=y
-- 
2.39.2

